docs/source/index.rst

```diff
@@ -92,6 +92,7 @@ NumPyro documentation
    examples/hmcecs
    examples/stein_bnn
    examples/stein_dmm
+   tutorials/other_samplers
```

Comment:
Maybe move this to introduction section.

Reply:
Changed in https://github.com/pyro-ppl/numpyro/pull/1842/commits/cba8f13603cbe2c209f6c493c9ef98a4d0e34916